showcase/sankey/voronoi.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import Sankey from 'sankey';
+
+const BLURRED_NODE_OPACITY = 0.8;
+const FOCUSED_NODE_OPACITY = 1;
+
+const nodes = [{name: 'a'}, {name: 'b'}, {name: 'c'}];
+const links = [
+  {source: 0, target: 1, value: 10},
+  {source: 0, target: 2, value: 20},
+  {source: 1, target: 2, value: 20}
+];
+
+export default class VoronoiSankeyExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeNode: null
+    };
+  }
+  render() {
+    const {activeNode} = this.state;
+
+    return (
+      <Sankey
+        nodes={nodes.map(d => ({
+          ...d,
+          opacity: activeNode && d.name === activeNode.name ? FOCUSED_NODE_OPACITY : BLURRED_NODE_OPACITY
+        }))}
+        links={links.map(d => ({...d}))}
```

Comment:
No unfortunately, because d3.sankey mutates the objects in the `links` array, and passing that array to the sankey component again causes an error. (I think it's just a PropTypes issue, but I didn't have time to track it down.)